babylon-to-espree/toAST.js

```diff
@@ -1,4 +1,5 @@
 var source;
+var t = require("babel-types");
```

Comment:
unfortunately, creating raw nodes this way removes `loc` information, which ESLint generally needs... hence the super-kludgy cloning that's been going on so far.